homeassistant/components/ecobee/sensor.py

```diff
@@ -19,12 +19,14 @@
         name="Temperature",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class="measurement",
```

Comment:
I was looking into making this change as well, and does anyone know if using the "measurment" string value or the `homeassistant.components.sensor.STATE_CLASS_MEASUREMENT` constant from the [sensor module](https://github.com/home-assistant/core/blob/dev/homeassistant/components/sensor/__init__.py#L100) is the preferred standard when updating integrations for statistics support?
I noticed at least some integrations are using the constant value like [NUT](https://github.com/home-assistant/core/blob/dev/homeassistant/components/nut/const.py#L71)

Reply:
Good call. Thanks!
While the descriptions I read earlier noted the string value. The existence of the constant, as well as the usage in a number of integrations leads me to the conclusion that that the constant is the right way to go.